init/arch_specs/eessi_arch_arm.spec

```diff
@@ -0,0 +1,6 @@
+# ARM CPU architecture specifications
+# Software path in EESSI 	| Vendor ID 	| List of defining CPU features
+"aarch64/arm/neoverse-n1"	"ARM"		"asimd"		# Ampere Altra
+"aarch64/arm/neoverse-n1"	""		"asimd"		# AWS Graviton2
+"aarch64/arm/neoverse-v1"	"ARM"		"asimd svei8mm" 
+"aarch64/arm/neoverse-v1"	""		"asimd svei8mm"	# AWS Graviton3
```

Comment:
do we want this in a more "formal" format like json/yaml?
(although that would add the risk of more replication of archspec)

Reply:
I would do that in a more normal situation, the problem with json/yaml format is that you need extra tools to read those and this should work in very minimal environments

Reply:
ack